miner/worker.go

```diff
@@ -213,8 +213,9 @@ type worker struct {
 	running int32 // The indicator whether the consensus engine is running or not.
 
 	// Test hooks
-	newTaskHook      func(*task) // Method to call upon receiving a new sealing task
-	fullTaskInterval func()      // Method to call before pushing the full sealing task
+	newTaskHook      func(*task)      // Method to call upon receiving a new sealing task
+	skipSealHook     func(*task) bool // Method to decide whether skipping the sealing.
+	fullTaskInterval func()           // Method to call before pushing the full sealing task
```

Comment:
This seems an odd name. Is `fullTaskInterval` only a test hook? If yes, please call it `fullTaskHook` to make it clear that it's a mock method, not a generally used one.